gremlin-go/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#    or more contributor license agreements.  See the NOTICE file
+#    distributed with this work for additional information
+#    regarding copyright ownership.  The ASF licenses this file
+#    to you under the Apache License, Version 2.0 (the
+#    "License"); you may not use this file except in compliance
+#    with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing,
+#    software distributed under the License is distributed on an
+#    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#    KIND, either express or implied.  See the License for the
+#    specific language governing permissions and limitations
+#    under the License.
+version: "3.4"
+services:
+  gremlin-server:
+    container_name: gremlin-server
+    image: tinkerpop/gremlin-server
+    ports:
+      - "8182:8182"
+  integration-tests:
+    container_name: integration-tests
+    image: golang:1.17
+    volumes:
+      - .:/go_app
+    environment:
+      - GREMLIN_SERVER_HOSTNAME=gremlin-server
+      - GREMLIN_SERVER_PORT=8182
+      - RUN_INTEGRATION_TESTS=true
+    working_dir: /go_app
+    command:
```

Comment:
Let's consider adding https://github.com/haveyoudebuggedit/gotestfmt, so it is easier to view test results?